From 14bb3b1ac022af33cac9952b2239fbccc08a256e Mon Sep 17 00:00:00 2001
From: Steve Dickson <SteveD@redhat.com>
Date: Thu, 26 Jun 2014 15:53:45 -0400
Subject: [fs] nfs: clean up PG_* flags

Message-id: <1403798077-28799-27-git-send-email-steved@redhat.com>
Patchwork-id: 84885
O-Subject: [RHEL7.1 PATCH 26/78] nfs: clean up PG_* flags
Bugzilla: 1113248
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>

From: Weston Andros Adamson <dros@primarydata.com>

Remove unused flags PG_NEED_COMMIT and PG_NEED_RESCHED.
Add comments describing how each flag is used.

Signed-off-by: Weston Andros Adamson <dros@primarydata.com>
Signed-off-by: Trond Myklebust <trond.myklebust@primarydata.com>
Upstream Id: 12c05792599ec57ebab33096b2c75b863dfe6ea4
Bugzilla: 1113248
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/linux/nfs_page.h b/include/linux/nfs_page.h
index c6a587f..eb2eb63 100644
--- a/include/linux/nfs_page.h
+++ b/include/linux/nfs_page.h
@@ -22,12 +22,10 @@
  * Valid flags for a dirty buffer
  */
 enum {
- PG_BUSY = 0,
- PG_MAPPED,
- PG_CLEAN,
- PG_NEED_COMMIT,
- PG_NEED_RESCHED,
- PG_COMMIT_TO_DS,
+ PG_BUSY = 0,  /* nfs_{un}lock_request */
+ PG_MAPPED,  /* page private set for buffered io */
+ PG_CLEAN,  /* write succeeded */
+ PG_COMMIT_TO_DS, /* used by pnfs layouts */
 };
 
 struct nfs_inode;
-- 
1.7.1